pages/contact.html

```diff
@@ -41,5 +41,18 @@
     </a>
 </div>
 </div>
+
+<div class="ui main text container">
+    <h1 class="ui header">School Contact Details</h1>
```

Comment:
The header is pretty close to the nav bar, might be worth adding a small bit of padding to it. For a quick fix, the following should cover it. This is inline CSS where you inject CSS into a tag, not best practice, but will be ok if we only need to do it once, if we notice this is a problem on all pages we can create our own CSS class for adding padding between the nav bar and the body of the page.
`<h1 class="ui dividing header" style="padding-top:8px">School Contact Details</h1>`

I also changed the header to a `dividing` header, this is a personal preference as it adds a bit more texture to the page, but worth a quick discussion with the team to settle on a style for all headers across the website.